gradle-plugin/src/main/kotlin/xyz/block/gradle/Flags.kt

```diff
@@ -36,31 +36,22 @@ internal val Settings.useLocalProtos: Boolean
   get() = providers.gradleProperty(LOCAL_PROTOS_ARTIFACTS)
     .getOrElse("false").toBoolean()
 
-val Project.isSandbagPublishingEnabled: Boolean
-  get() = providers.gradleProperty(IS_SANDBAG_PUBLISHING)
-    .getOrElse("false").toBoolean()
-
-val Settings.isArtifactPublishingEnabled: Boolean
-  get() = providers.gradleProperty(IS_SANDBAG_PUBLISHING)
-    .getOrElse("false").toBoolean()
-
-val Project.sandbagHashFile: File
-  get() = File(rootDir, providers.gradleProperty(SANDBAG_HASH_FILE).get())
+val Project.artifactHashFile: File
```

Comment:
Is there a reason we're using `java.io` instead of the Gradle API (like `RegularFile`)?